Add opencart and openvvc files (#150)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/openvvc.md

---
name: OpenVVC
category: Video

> description: OpenCart is an online store management system developed. It is PHP-based, using a MySQLi or PostgreSQL database and HTML components.

Author: This is opencart's description

download_url: https://github.com/OpenVVC/OpenVVC/tags
works_on_arm: true
supported_minimum_version:
    version_number: 1.0.0
    release_date: 20/12/2021

optional_info:
    homepage_url: https://openvvc.github.io/
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content:
        partner_content:
        official_docs: https://github.com/OpenVVC/OpenVVC?tab=readme-ov-file#compiling-openvvc
    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum:
    release_notes__recommended_minimum:

>     other_info: Linux/ARM64 release notes are not available. Installation and testing are done via the tar.

Author: ```suggestion
    other_info: Linux/ARM64 release notes are not available. Installation and testing were done using released tar files.
```
